library/hylang

```diff
@@ -12,13 +12,13 @@ Tags: 1.0.0-python3.13-bullseye, 1.0-python3.13-bullseye, 1-python3.13-bullseye,
 Architectures: amd64, arm32v7, arm64v8, i386
 File: Dockerfile.python3.13-bullseye
 
-Tags: 1.0.0-python3.13-alpine3.20, 1.0-python3.13-alpine3.20, 1-python3.13-alpine3.20, python3.13-alpine3.20, 1.0.0-alpine3.20, 1.0-alpine3.20, 1-alpine3.20, alpine3.20, 1.0.0-python3.13-alpine, 1.0-python3.13-alpine, 1-python3.13-alpine, python3.13-alpine, 1.0.0-alpine, 1.0-alpine, 1-alpine, alpine
+Tags: 1.0.0-python3.13-alpine3.21, 1.0-python3.13-alpine3.21, 1-python3.13-alpine3.21, python3.13-alpine3.21, 1.0.0-alpine3.21, 1.0-alpine3.21, 1-alpine3.21, alpine3.21
 Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le, s390x
-File: Dockerfile.python3.13-alpine3.20
+File: Dockerfile.python3.13-alpine3.21
 
-Tags: 1.0.0-python3.13-alpine3.19, 1.0-python3.13-alpine3.19, 1-python3.13-alpine3.19, python3.13-alpine3.19, 1.0.0-alpine3.19, 1.0-alpine3.19, 1-alpine3.19, alpine3.19
+Tags: 1.0.0-python3.13-alpine3.20, 1.0-python3.13-alpine3.20, 1-python3.13-alpine3.20, python3.13-alpine3.20, 1.0.0-alpine3.20, 1.0-alpine3.20, 1-alpine3.20, alpine3.20, 1.0.0-python3.13-alpine, 1.0-python3.13-alpine, 1-python3.13-alpine, python3.13-alpine, 1.0.0-alpine, 1.0-alpine, 1-alpine, alpine
```

Comment:
Did you want to move latest Alpine tags to 3.21 or keep it at 3.20?

Reply:
:facepalm: